### PR TITLE
fix: update grpc packages and rust crate

### PR DESCRIPTION
**Pull request by tiagolobocastro**
fix: update run_script crate
    
    Updates crate to newer version to remove a dependent which was flagged as
    vulnerability by dependabot.
    
    Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

---

    fix(test/grpc): update packages
    
    This should help silence some of the security alerts
    (not that it matters too much since this is test code..)
    
    Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

### CODE REVIEW

**Comment by tiagolobocastro** [comment]
bors try

**Comment by tiagolobocastro** [comment]
bors try

File: Cargo.lock
Comment (hrudaya21) [inline Cargo.lock:1]: Do We need to commit `cargo.lock` changes ? It is auto generated from `toml` file right ?

**Review by dsharma-dc — APPROVED** [review]

File: Cargo.lock
Comment (tiagolobocastro) [reply Cargo.lock:1]: Yes cargo.lock must be committed otherwise you don't get reproducible builds

**Review by hrudaya21 — APPROVED** [review]

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**